libkatana_python_native/include/katana/python/CythonIntegration.h

```diff
@@ -132,16 +132,17 @@ namespace katana {
 
 // Add Cython classes here as needed. Remove them when they are moved to
 // pybind11.
-CYTHON_REFERENCE_SUPPORT(katana::PropertyGraph, "katana.local", "Graph");
-CYTHON_REFERENCE_SUPPORT(katana::TxnContext, "katana.local", "TxnContext");
+//CYTHON_REFERENCE_SUPPORT(katana::PropertyGraph, "katana.local", "Graph");
```

Comment:
Do you still need to commented out line?

Reply:
I was keeping it as an example: I'll add "For example" to the comment to make that clear.